web/api/v1/api_test.go

```diff
@@ -3299,7 +3224,7 @@ func BenchmarkRespond(b *testing.B) {
 	b.ResetTimer()
 	api := API{}
```

Comment:
This benchmark is failing right now, you will need to install the json codec after this line.

Reply:
Good catch, I've fixed this.